docs/contribute/core.md

```diff
@@ -64,6 +64,33 @@ The following is the `.gitignore` file in the `asdf-vm/asdf` repository. We igno
 
 @[code](../../.gitignore)
 
+### `.git-blame-ignore-revs`
+
+`asdf` uses a `.git-blame-ignore-revs` to reduce noise when running a blame.
+
+Use the file with `git blame` like so:
+
+```sh:no-line-numbers
+git blame --ignore-revs-file .git-blame-ignore-revs ./test/install_command.bats
+```
+
+Optionally, configure to use the file on every invocation of `blame` without manually supplying it:
+
+```sh:no-line-numbers
+git config blame.ignoreRevsFile .git-blame-ignore-revs
+```
+
+If using an IDE, configure it to enhance the default experience. For example, when using VSCode (with GitLens) configure it like so:
+
+```json:no-line-numbers
+{
+  "gitlens.advanced.blame.customArguments": [
+    "--ignore-revs-file",
+    ".git-blame-ignore-revs"
+  ]
+}
+```
```

Comment:
is this a file supposed to be in `.vscode/settings.json`? Can you say which file these settings should go in?